ddcz/forms/commentForms.py

```diff
@@ -0,0 +1,9 @@
+from django import forms
+from django.http import HttpResponse
+from ..models.used.discussion import Phorum
+
+
+class PhorumCommentForm(forms.Form):
+    text = forms.CharField(label='', widget=forms.Textarea(attrs={'class': 'comment__textarea'}))
+    username = forms.HiddenInput()
```

Comment:
If I am not mistaken, you have created a security hole. If I'll change the value of the username in the hidden field, I can comment as someone else.

Please remove and use user info from the session.

Reply:
Nevermind, not a security hole—but why is the username there?

Reply:
Fixed in new version